### PR TITLE
Address issue of python36-libvirt for kvm/SG CentOS users

### PR DESCRIPTION
**Pull request by rhtyd**
From https://github.com/apache/cloudstack/pull/4124

### CODE REVIEW

**Comment by rhtyd** [comment]
request docbuild

**Comment by NuxRo** [comment]
The instruction "yum install python36-libvirt" will become unnecessary once we build new rpms with the correct deps, but better mention it than not.

**Comment by rhtyd** [comment]
The PR is to immediately update the 4.14.0.0 docs website. Yes, if we decide to 4.14.1.0 soon then the explicit command won't be necessary.

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -46,6 +46,12 @@ The full list of new features can be found in the project release notes at http:
 
 Apache CloudStack powers numerous elastic Cloud computing services, including solutions that have ranked as Gartner Magic Quadrant leaders. Highlighted in the Forrester Q4 2017 Enterprise Open Source Cloud Adoption report, Apache CloudStack "sits beneath hundreds of service provider clouds", including Fortune 5 multinational corporations. A list of known Apache CloudStack users are available at http://cloudstack.apache.org/users.html
 
+Libvirt Python Dependency on KVM and CentOS
+===========================================
+
+For CentOS users using the security groups feature on KVM need to install
+epel-release and python36-libvirt package.
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:53]:
```suggestion
For CentOS users using the security groups feature on KVM it is needed to install the epel-release and python36-libvirt packages.
```

**Comment by rhtyd** [comment]
Thanks I'll merge, re-tag and re-publish the docs website.

**Pull request merged**